pyproject.toml

```diff
@@ -53,6 +52,10 @@ jupyter = [
     "ipywidgets",
 ]
 
+graphics = [
+    "ansys-tools-visualization-interface>=0.2.6",
+]
+
```

Comment:
We should handle inside the code any attempted imports @AlejandroFernandezLuces and throw an error with users instructions for installing PyMAPDL with the graphics target.

Reply:
It's already handled:

https://github.com/ansys/pymapdl/blob/16508b74298c96638093d257e503cc5473995a17/src/ansys/mapdl/core/mapdl_extended.py#L440-L444

I think @germa89 implemented the dependencies as mandatory on purpose, I prefer to have his feedback before changing all messages to install the optional target

Reply:
@germa89 - you should provide users instructions on how to install the graphics target. Not what dependencies are missing. A user would like to "know" how to solve this issue basically - i.e. what command to run.

Reply:
@RobPasMue you mean change the exception message to something like:

```
             "Using the keyword argument 'vtk' requires having 'ansys-tools-visualization_interface' installed. "
             "You  can install this using `pip install ansys-mapdl-core[graphics]`."
```

???

Reply:
Correct =)

Reply:
If that's okay with you I'll change the messages

Reply:
It is OK with me giving more info the exception message :)